docs/source/routing/performance/caching/entity.mdx

```diff
@@ -162,6 +162,7 @@ For example:
 # Enable entity caching globally
 preview_entity_cache:
   enabled: true
+  expose_keys_in_context: true # Optional, it will expose cache keys in the context in order to use it in coprocessors or Rhai
```

Comment:
This needs more description in the docs for what it does and the related workflow.

Reply:
I wanted to have more feedback from beta testers to officially document it. What do you think ?

Reply:
Fair enough.